lib/models/storage-event.js

```diff
@@ -22,11 +22,7 @@ var StorageEvent = new mongoose.Schema({
   user: {
     type: String, 
     ref: 'User',
-    required: true,
```

Comment:
This should remain as true. And email validation should be consistent across all models, why removed the validation?